### PR TITLE
Fixes #21523: Initialisation of campaign blocks rudder

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/21523

### CODE REVIEW

File: webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala [hunk 1]
```diff
@@ -2503,14 +2508,23 @@ object RudderConfig extends Loggable {
...
+    new RuntimeException(s"Error during initialization of campaign repository: " + err.fullMsg)
+  )
 
   val mainCampaignService  = new MainCampaignService(campaignEventRepo, campaignRepo, uuidGen)
-  MainCampaignService.start(mainCampaignService).runNow
-
   lazy val jsonReportsAnalyzer = JSONReportsAnalyser(reportsRepository, propertyRepository)
-  jsonReportsAnalyzer.start().forkDaemon
+
+  println(s"********** here 1 ")
```
Comment (amousset) [inline webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala:2519]: look like debug prints

**Comment by amousset** [comment]
LGTM apart form the debug print

**Comment by fanf** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Comment by fanf** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala (hunk 1 above)
Comment (fanf) [reply webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala]: :+1:

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**